core/src/main/java/nl/nn/adapterframework/core/PipeRunResult.java

```diff
@@ -37,7 +37,7 @@
  * @see nl.nn.adapterframework.pipes.AbstractPipe#doPipe
  * @see nl.nn.adapterframework.pipes.AbstractPipe#findForward
  */
-public class PipeRunResult {
+public class PipeRunResult implements GenericSenderResult {
```

Comment:
Ik denk dat het verstandig is het pipe result en sender result gescheiden te houden van elkaar.